cypress/e2e/buy-tshirt.cy.ts

```diff
@@ -11,12 +12,6 @@ const addressStepPage = new AddressStepPage();
 const shippingStepPage = new ShippingStepPage();
 const paymentStepPage = new PaymentStepPage();
 
-const validationMessage = "Your order on My Store is complete.";
-const userId = {
-  email: "aperdomobo@gmail.com",
-  password: "WorkshopProtractor",
-};
-
 describe("Buy a t-shirt", () => {
   it("then the t-shirt should be bought", () => {
```

Comment:
what we want, is to you initialize the page objects of this test in a similar way than in the `cypress/e2e/dresses-list.cy.ts` file

Reply:
All changes done